src/main/java/gov/nasa/pds/harvest/util/CloseUtils.java

```diff
@@ -24,7 +25,22 @@ public static void close(Closeable cl)
             // Ignore
         }
     }
+
     
+    public static void close(Stream<?> cl)
+    {
+        if(cl == null) return;
+        
+        try
+        {
+            cl.close();
+        }
+        catch(Exception ex)
+        {
+            // Ignore
```

Comment:
I think it is better to at least log the exception, so that we know there was a problem while closing the stream.

Reply:
Added warning